### PR TITLE
allow only supplementalGroups greater 0

### PR DESCRIPTION
**Pull request by smnmtzgr**

### CODE REVIEW

**Comment by realshuting** [comment]
@smnmtzgr - looks good!

Logged this issue to update tests https://github.com/kyverno/policies/issues/58.

**Comment by chipzoller** [comment]
If the minimum version of Kyverno needed to support this is 1.3.6, we need to add the relevant annotation as seen [here](https://github.com/kyverno/policies/blob/b6aa5dbdca71cf7ac4e0cd09d97b9fc72e067212/pod-security/baseline/restrict-apparmor-profiles/restrict-apparmor-profiles.yaml#L10). This also needs to be held until 1.3.6 releases.

**Comment by smnmtzgr** [comment]
@chipzoller should I add this annotation?

**Comment by chipzoller** [comment]
Yes, let's go ahead and add this annotation now that 1.3.6 is released.

**Comment by smnmtzgr** [comment]
@chipzoller / @realshuting I have added the annotation. Can you check it again and if okay please merge.

File: pod-security/restricted/require-non-root-groups/require-non-root-groups.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ metadata:
   annotations:
     policies.kyverno.io/category: Pod Security Standards (Restricted)
     policies.kyverno.io/severity: medium
+    policies.kyverno.io/minversion: 1.6.0
```
Comment (realshuting) [inline pod-security/restricted/require-non-root-groups/require-non-root-groups.yaml:8]: Hi @smnmtzgr - the minimum version should be 1.3.6.

**Review by realshuting — CHANGES_REQUESTED** [review]

File: pod-security/restricted/require-non-root-groups/require-non-root-groups.yaml (hunk 1 above)
Comment (smnmtzgr) [reply pod-security/restricted/require-non-root-groups/require-non-root-groups.yaml]: sorry fixed...

**Review by realshuting — APPROVED** [review]

**Pull request merged**